<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Add new client:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>port:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remove">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>411</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_ip">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="txt_port">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>71</y>
      <width>401</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
